source/importing-css-assets-and-javascript/index.html.md.erb

```diff
@@ -72,6 +72,23 @@ If you want to override GOV.UK Frontend's styles with your own styles, `@import`
 
 If your project uses GOV.UK Frontend toolkit, GOV.UK Template or GOV.UK Elements, you can [configure GOV.UK Frontend to work with them](/compatibility-mode/).
 
+### Silence deprecation warnings in Dart Sass
+
+If you are using Dart Sass v1.33.0 or greater, you may see deprecation warnings when compiling your Sass:
+
+```
+DEPRECATION WARNING: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.
+```
+
+We are currently unable to fix these deprecation warnings without breaking support for LibSass.
+
+You can however silence the warnings. Ensure you are using Dart Sass 1.35.0 or greater and then:
+
+- if you are calling the sass compiler from the command line, pass [the `--quiet-deps` flag](https://sass-lang.com/documentation/cli/dart-sass#quiet-deps)
+- if you are using the JavaScript API, include [`quietDeps: true`](https://sass-lang.com/documentation/js-api#quietdeps) in the `options` object
+
+This will only silence deprecation warnings that are caused by dependencies, including GOV.UK Frontend. You will still see deprecation warnings if you are using `/` for division in your own Sass code.
```

Comment:
Suggesting we cut the dependencies-related sentence from here, if we're now mentioning it in line 85 instead:

```suggestion
You will still see deprecation warnings if you're using `/` for division in your own Sass code.
```